src/main/java/baritone/utils/schematic/litematica/LitematicaBlockStatePaletteLinear.java

```diff
@@ -0,0 +1,112 @@
+package baritone.utils.schematic.litematica;
+
+import javax.annotation.Nullable;
+import net.minecraft.block.BlockState;
+import net.minecraft.nbt.CompoundNBT;
+import net.minecraft.nbt.ListNBT;
+import net.minecraft.nbt.NBTUtil;
+
+public class LitematicaBlockStatePaletteLinear implements ILitematicaBlockStatePalette
```

Comment:
Same shit

Reply:
Small changes are required to this class. However this can't be an extension of its minecraft counterpart as the aforementioned makes reference to the non public interface IBlockStateContainer